src/index.tsx

```diff
@@ -53,6 +55,16 @@ const memoryOnlyPrecomputedFlagsStore: IConfigurationStore<PrecomputedFlag> =
 const memoryOnlyPrecomputedBanditsStore: IConfigurationStore<IObfuscatedPrecomputedBandit> =
   new MemoryOnlyConfigurationStore();
 
+/**
+ * Builds a storage key suffix from an API key.
+ * @param apiKey - The API key to build the suffix from
+ * @returns A string suffix for storage keys
+ */
+function buildStorageKeySuffix(apiKey: string): string {
```

Comment:
this is leaky and has led to collisions in unimaginatively written tests (ex: `const apiKeys = ['SDK_KEY_1', 'SDK_KEY_2', 'SDK_KEY_3']`).
May I suggest hashing the API key? Can still take a substring of that.

Reply:
Oh interesting observation! This works fine for real SDK keys since they are strings of random characters. An additional benefit of not hashing and using the original SDK key in the storage key is that it's easier to find the record that corresponds with the client when debugging.

But I see your point with writing tests and developer QOL in that regard. Will add a md5 hasher dependency and implement here